__tests__/integration/melt-tokens.test.js

```diff
@@ -291,18 +331,25 @@ describe('melt tokens', () => {
       .send({
         token: tokenA.uid,
         address: await wallet1.getAddressAt(1),
-        amount: 100
+        amount: 110
```

Comment:
This test was supposed to melt a value not multiple of 100, for instance  check that 110 TKA melted would give back only 1 HTR.